libs/ballot-interpreter/src/hmpb-rust/interpret.rs

```diff
@@ -313,6 +323,24 @@ pub fn interpret_ballot_card(
         None => ImageDebugWriter::disabled(),
     };
 
+    [
+        (SIDE_A_LABEL, &side_a, &mut side_a_debug),
+        (SIDE_B_LABEL, &side_b, &mut side_b_debug),
+    ]
+    .par_iter_mut()
+    .map(|(label, side, debug)| {
+        let streaks = detect_vertical_streaks(&side.image, side.threshold, debug);
+        if streaks.is_empty() {
+            Ok(())
+        } else {
+            Err(Error::VerticalStreaksDetected {
+                label: (**label).to_string(),
```

Comment:
I don't think the derefs are needed.
```suggestion
                label: label.to_string(),
```

Reply:
I didn't have them originally but clippy requested them

Reply:
Made it a tad better: 5bf0b0b41

Reply:
I'm surprised that clippy isn't complaining for me. What's your Rust version? Here's mine:

```
❯ rustc --version
rustc 1.80.0 (051478957 2024-07-21)
```

Reply:
Same:
```
$ rustc --version
rustc 1.80.0 (051478957 2024-07-21)
```

Reply:
And now I do see it 🤷 

```
error: calling `to_string` on `&&str`
   --> libs/ballot-interpreter/src/hmpb-rust/interpret.rs:337:24
    |
337 |                 label: label.to_string(),
    |                        ^^^^^^^^^^^^^^^^^ help: try dereferencing the receiver: `(*label).to_string()`
    |
    = help: `&str` implements `ToString` through a slower blanket impl, but `str` has a fast specialization of `ToString`
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#inefficient_to_string
```